Buildings/Controls/OBC/CDL/Integers/Greater.mo

```diff
@@ -11,8 +18,108 @@ block Greater
     "Connector of Boolean output signal"
     annotation (Placement(transformation(extent={{100,-20},{140,20}})));
 
+protected
+  final parameter Boolean have_hysteresis=h >= 1E-10
```

Comment:
`h` is an `Integer`, so just check for `h > 0`. (I am surprised that this gives not a warning.) Please also change in the other new blocks.